tests/console/zypper_lr.pm

```diff
@@ -160,11 +160,21 @@ __END' . "\n";
             # This is where we verify base product repos for openSUSE, SLES, SLED,
             # and HA
             if (check_var('FLAVOR', 'Server-DVD')) {
+                my $uri = "cd:///";
+                if (check_var("BACKEND", "ipmi") || check_var("BACKEND", "generalhw")) {
+                    $uri = "http[s]*://.*suse";
```

Comment:
please note that Beijing office uses different repos. Are you still certain you can cover all cases? :)